LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DefaultLoRaDataRequestHandler.cs

```diff
@@ -289,7 +289,7 @@ public async Task<LoRaDeviceRequestProcessResult> ProcessRequestAsync(LoRaReques
 
                             if (decodePayloadResult.CloudToDeviceMessage != null)
                             {
-                                if (string.IsNullOrEmpty(decodePayloadResult.CloudToDeviceMessage.DevEUI) || string.Equals(loRaDevice.DevEUI, decodePayloadResult.CloudToDeviceMessage.DevEUI, StringComparison.OrdinalIgnoreCase))
+                                if (decodePayloadResult.CloudToDeviceMessage.DevEUI == default || loRaDevice.DevEUI == decodePayloadResult.CloudToDeviceMessage.DevEUI)
```

Comment:
`is null` reads clearer here to mean missing:

```suggestion
                                if (decodePayloadResult.CloudToDeviceMessage.DevEUI is null || loRaDevice.DevEUI == decodePayloadResult.CloudToDeviceMessage.DevEUI)
```